src/sentry/web/urls.py

```diff
@@ -322,6 +322,9 @@ def init_all_applications():
     url(r'^api/(?P<project_id>[\w_-]+)/crossdomain\.xml$', api.crossdomain_xml,
         name='sentry-api-crossdomain-xml'),
 
+    # plugins
+    url(r'', include('sentry.plugins.base.urls')),
```

Comment:
I'm not totally sure of the use case here, but I wouldn't let plugins own the root namespace.

I'd give them something like: `url(r'^api/hooks/', include('sentry.plugins.base.urls'))` or something akin to that.